packages/agent/package-lock.json

```diff
@@ -0,0 +1,11140 @@
+{
```

Comment:
@mehdibalouchi can you remove this file? Its [been argued](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5) that lock files should be checked-in, though its [commonly agreed upon](https://stackoverflow.com/a/44635341/1568394) that lock files should be ignored in libraries.